Dalamud/Storage/ReliableFileStorage.cs

```diff
@@ -91,8 +92,9 @@ public bool Exists(string path, Guid containerId = default)
     /// <param name="path">Path to write to.</param>
     /// <param name="contents">The contents of the file.</param>
     /// <param name="containerId">Container to write to.</param>
-    public void WriteAllText(string path, string? contents, Guid containerId = default)
```

Comment:
Isn't that here a breaking change, as the class is public and could have been used since it was released with Api 9?

Reply:
This isn't a plugin-exposed service. The entire interface should probably be `internal`, but eh.